.github/workflows/update-test-rules.yml

```diff
@@ -45,12 +45,12 @@ jobs:
 
       - name: Get PR branch
         uses: alessbell/pull-request-comment-branch@v1.1 # Fork of xt0rted/pull-request-comment-branch, see https://github.com/xt0rted/pull-request-comment-branch/issues/322
-        id: comment-branch
+        id: comment_branch
```

Comment:
Changed it here for consistency. `_` preferred so that it's considered a single word in most contexts.